Detectors/TOF/workflow/src/TOFRawWriterSpec.cxx

```diff
@@ -86,6 +100,19 @@ void RawWriter::run(ProcessingContext& pc)
   encoder.flush();
   encoder.close();
 
+  // create configuration file for rawreader
+  std::ofstream fconfig;
+  fconfig.open("TOFraw.cfg");
+  fconfig << "[defaults]\n";
+  fconfig << "dataOrigin = TOF\n";
+  fconfig << "dataDescription = RAWDATA\n\n";
+
+  for (int icru = 0; icru < encoder.getNCRU(); icru++) {
+    fconfig << "[input-" << icru << "]\n";
+    fconfig << "filePath = " << mOutDirName << "/cru" << std::setfill('0') << std::setw(2) << icru << mOutFileName << "\n\n";
```

Comment:
... and instead add explicitly for every chunk
````
dataOrigin = TOF
dataDescription = RAWDATA
````

Reply:
> ... and instead add explicitly for every chunk
> 
> ```
> dataOrigin = TOF
> dataDescription = RAWDATA
> ```

I also replaced "input-" with "input-TOF-"